packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -1672,22 +1678,32 @@ class InteractiveGraph extends React.Component<Props, State> {
         this.onChange({graph: graph});
     };
 
-    getUserInput: () => PerseusGraphType = () => {
-        return InteractiveGraph.getUserInputFromProps(this.props);
-    };
+    getUserInput: () => PerseusGraphType = () =>
+        this.mafsRef.current ??
+        InteractiveGraph.getUserInputFromProps(this.props);
 
-    simpleValidate: (rubric: Rubric) => PerseusScore = (rubric) => {
-        return InteractiveGraph.validate(this.getUserInput(), rubric, this);
-    };
+    simpleValidate: (rubric: Rubric) => PerseusScore = (rubric) =>
+        InteractiveGraph.validate(this.getUserInput(), rubric, this);
```

Comment:
I don't think this way of scoring the graph is going to work. `InteractiveGraph.validate` looks pretty entangled with our existing graph code, so I'd prefer to rethink how we do scoring as part of the Mafs rewrite.

Could we return a dummy value here for now?
```suggestion
        ({type: "invalid"}); // TODO: implement scoring
```

Reply:
Actually, `validate` is one of the only pure functions in the class. Grading works well, now that I've fixed the mafsRef.